pages/_en-US/twilightmenu/faq.md

```diff
@@ -35,12 +35,14 @@ You need to have a cheat DB in the form of a `usrcheat.dat` file in the `sd:/_nd
 
 Alternatively, you can use [r4cce](http://hp.vector.co.jp/authors/VA013928/soft_en.html) to create your own cheat DB.
 
-#### How do I show a custom picture on the top screen of the DSi theme?
+#### How do I show a custom picture on the top screen of the DSi theme? Can I hide it instead?
 A random `.png` image in `sd:/_nds/TWiLightMenu/dsimenu/photos/` will be shown each time the menu is loaded. If there are no applicable images, screenshots taken by nds-bootstrap will be used instead.
 
 - The images(s) must be no bigger than 208x156
 - If you have errors, it's most likely an error with the image size. Please use [tinypng](https://tinypng.com) to reduce the size
 
+As for hiding the picture, you need to edit the `theme.ini` file found in `sd:/_nds\TWiLightMenu\dsimenu\themes\[folder of the skin you're using]\`. Open the file with a text editor change the line `RenderPhoto` from `1` to `0`, then save the file.
```

Comment:
Missing a comma after editor
`Open the file with a text editor, change the line ...`